### ISSUE TITLE
Extension keeps asking to select a note location

### ISSUE THREAD

**Issue by hayksaryan** [dionmunk/vscode-notes#54]
## Details
OS: Ubuntu 24.04

## What have I tried?
- Reloading
- Restarting
- Selecting different sub-folders of Home and the project folder
- Reinstalling the extension

**Comment by hayksaryan** [comment]
Realized this was rather a UX issue and not a bug. Maybe we need to change the default view if the folder is already selected.

**Comment by darylshy** [comment]
> ## Details
> OS: Ubuntu 24.04
> 
> ## What have I tried?
> * Reloading
> * Restarting
> * Selecting different sub-folders of Home and the project folder
> * Reinstalling the extension

I'm here for the exact same reason. 

https://github.com/dionmunk/vscode-notes/assets/5904383/175d3cc1-cba4-4dd5-a039-b9be0b31fd42

![Screenshot 2024-04-01 at 12 48 30 PM](https://github.com/dionmunk/vscode-notes/assets/5904383/16a25569-31b3-41f2-89f4-4c7c3176f467)

**Comment by ggat** [comment]
@hayksaryan thanks! I would never guess that by myself.

Nevertheless, it's so bad that I was going to uninstall and search for another ext.

**Comment by hayksaryan** [comment]
> @hayksaryan thanks! I would never guess that by myself.
> 
> Nevertheless, it's so bad that I was going to uninstall and search for another ext.

I'd already done that for you. But then decided I'm not going to succumb to this extension :smile:

**Comment by pablocubico** [comment]
Hey @hayksaryan but are you able to see the notes list as shown in the Extension install page?:

![image](https://github.com/dionmunk/vscode-notes/assets/340766/8b312094-58a0-4f68-8fa6-8913dedbc5dc)

I think this is a bug, not bad UX, I'm facing the same issue. Hope it gets fixed!

**Comment by hayksaryan** [comment]
@pablocubico Yes, I see the list. Once you create at least one note the list appears in the sidebar.

**Comment by scypher00** [comment]
I have the same issue, **notes are not listed on the sidebar, even after I create a new note**. 
If I do Ctrl+Shift+P > Notes: List notes, they show up on the command drop down. 

I am running on Remote Development mode, and the Notes.notesLocation is a path on the remote machine (I do see the new notes saved there).  

My VSCode versions are:
Version: 1.87.2 (system setup)
Commit: 863d2581ecda6849923a2118d93a088b0745d9d6
Date: 2024-03-08T15:20:17.278Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by scypher00** [comment]
uninstalling and re-installing fixed it

**Comment by nyxb** [comment]
on new vscode with mac sonoma it doesnt work anymore

**Comment by dionmunk** [comment]
Thank you all for pointing out this issue. I am sure it has been a problem for many people. I'll work on a fix as soon as possible.

**Comment by eusebiedev** [comment]
Hi! I'd love to use this extension. Unfortunately, it still keeps asking for a storage location. 
- also, when renaming a note, the note disappears from the list..  and again asks for a storage location.
Best.

**Comment by edgarandresfp** [comment]
If anyone else is facing the same issue, I found a workaround that might help:

- Select your desired storage location.
- Close VS Code.
- Create a test.md file in the folder you selected as the storage location.
- Open VS Code again.

Your test.md note should appear, and everything should work as expected.

**Comment by svharsdorf** [comment]
> Realized this was rather a UX issue and not a bug. Maybe we need to change the default view if the folder is already selected.

Yeah, that took me a good second. 
I kept selecting a new location, then get prompted to refresh the window, then come back to "please select a location".

I suggest that if it has a location assigned and no notes yet, it displays a "New Note" button, as that would be the logical next step.

**Comment by MathieuBrillard** [comment]
Hello, I was using this extension for a long time now, but I started having this bug since a few days...

I pretty much tried anything I could (disabled/re-enabled the ext, uninstalled/reinstalled, changed note location folder and its permissions...), but nothing resolved the bug so far.

The only workaround I found is to use `Ctrl+Shift+P > Notes: List notes` as suggested by @scypher00 

It would be great if this was solved, because this extension is really helpful !
Keep up the good work :)

**Comment by dionmunk** [comment]
I will be making some updates to the extension soon, and I will fix this issue with the upcoming release. I appreciate all of your feedback!

**Comment by dionmunk** [comment]
Resolved by https://github.com/dionmunk/vscode-notes/pull/64.
